bin/build_sdb.py

```diff
@@ -67,6 +79,11 @@ def handle_args():
         help="If to build an SDB with unlabeled (audio only) samples - "
         "typically used for building noise augmentation corpora",
     )
+    parser.add_argument(
+        "--augment",
+        action='append',
```

Comment:
nit: mixing double and single quotes in this file

Reply:
Will fix it soon with the next PR on this file... :-)